### ISSUE TITLE
Super admin bug in Templates

### ISSUE THREAD

**Issue by vyruss** [DMPRoadmap/roadmap#106]
Commit https://github.com/DMPRoadmap/roadmap/commit/6dec11c32647a35d85fab017cb5ed5aa15b57c56#diff-939d88c8ca92b85897ad4f7b86ea7acf breaks `app/admin/dmptemplate.rb` for some reason (missing `%i` ?) which results in a 404 page when clicking the super admin: Templates link. 
```ruby
    action_item(:edit) do
        link_to(I18n.t('helpers.settings.title'), settings_admin_dmptemplate_path(resource.id))
    end
```

This may be also broken in https://github.com/DMPRoadmap/roadmap/blob/development/app/admin/template.rb

This bit of code seems to work:
```ruby
    action_item only: %i(edit) do
        link_to(I18n.t('helpers.settings.title'), settings_admin_dmptemplate_path(resource.id))
    end
```

**Comment by vyruss** [comment]
Blocked by #131 #132

**Comment by vyruss** [comment]
Closed by #131 #132
